scripts/apps/authoring/authoring/directives/ItemAssociationDirective.ts

```diff
@@ -149,3 +141,44 @@ export function ItemAssociationDirective(renditions) {
         },
     };
 }
+
+export function isAllowedMediaType(scope, event) {
+    const {allowAudio, allowPicture, allowVideo} = scope;
+    const VALID_MEDIA_TYPES = ['Files'];
```

Comment:
What is the significance of a string `'Files'`? Is it an arbitrary string or something the browser adds into the data transfer object? If it's not a browser thing, extract it to constants across the codebase.

Reply:
It's being added by the browser itself to differentiate between the type of data dropped eg: text/plain, text/uri-list, Files, etc.